src/applications/facility-locator/tests/e2e/search.cypress.spec.js

```diff
@@ -1,18 +1,36 @@
+import path from 'path';
+
 describe('Facility Search', () => {
+  before(() => {
+    cy.syncFixtures({
+      constants: path.join(__dirname, '..', '..', 'constants'),
+    });
+  });
+
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/v0/feature_toggles?*', []);
+    cy.route('GET', '/v0/maintenance_windows', []);
+    cy.route('GET', '/v0/facilities/va?*', 'fx:constants/mock-facility-data');
+    cy.route('GET', '/geocoding/**/*', 'fx:constants/mock-geocoding-data');
```

Comment:
I was just trying a bunch of things to get the tests passing on a prod build. So I know that stubbing the geocoding request isn't necessary, but I left it in in case it ends up being useful to your team.